lte/gateway/python/magma/mobilityd/ip_address_man.py

```diff
@@ -340,13 +368,24 @@ def alloc_ip_address(self, sid: str) -> Tuple[ip_address, int]:
                 logging.error(error_msg)
                 raise DuplicateIPAssignmentError(error_msg)
 
-            self.ip_state_map.add_ip_to_state(ip_desc.ip, ip_desc, IPState.ALLOCATED)
+            self.ip_state_map.add_ip_to_state(ip_desc.ip, ip_desc,
+                                              IPState.ALLOCATED)
             self.sid_ips_map[sid] = ip_desc
```

Comment:
We're using different types of store to hold IPv4 and IPv6 address, for IPv6 we hold IID in allocated_iid and session prefix in sid_session_prefix_allocated, in IPv4 we hold the full IPv4 address.

Reply:
right, but I thought we will use the map in follow up patches for extending IPv6 support, so it would be better to improve the key definition now then later.